tidb-distributed-execution-framework.md

```diff
@@ -0,0 +1,94 @@
+---
+title: TiDB Backend Task Distributed Execution Framework
+summary: Learn the use cases, limitations, usage, and implementation principles of the TiDB backend task distributed execution framework.
+---
+
+# TiDB Backend Task Distributed Execution Framework
+
+> **Warning:**
+>
+> This feature is an experimental feature. It is not recommended to use it in production environments.
+
+TiDB adopts a computing-storage separation architecture with excellent scalability and elasticity. Starting from v7.1.0, TiDB introduces a backend task distributed execution framework to further leverage the resource advantages of the distributed architecture. The goal of this framework is to implement unified scheduling and distributed execution of all backend tasks, and to provide unified resource management capabilities for the backend tasks, providing resource management capabilities for both the overall and individual backend tasks to better meet users' expectations for resource usage.
+
+This document describes the use cases, limitations, usage, and implementation principles of the TiDB backend task distributed execution framework.
+
+> **Note:**
+>
+> This framework does not support the distributed execution of SQL queries.
+
+## Use cases and limitations
+
+In a database management system, in addition to the core transactional processing (TP) and analytical processing (AP) workloads, there are other important tasks, such as DDL operations, Load Data, TTL, Analyze, and Backup/Restore, which are called **backend tasks**. These backend tasks need to process a large amount of data in database objects (tables), so they typically have the following characteristics:
+
+- Need to process all data in a schema or a database object (table).
+- Might need to be executed periodically, but at a low frequency.
+- If the resources are not properly controlled, they are prone to affect TP and AP tasks, lowering the database service quality.
+
+Enabling the TiDB backend task distributed execution framework can solve the above problems and has the following three advantages:
+
+- The framework provides unified capabilities for high scalability, high availability, and high performance.
+- The framework supports distributed execution of backend tasks, which can flexibly schedule the available computing resources of the entire TiDB cluster, thereby better utilizing the computing resources in the TiDB cluster.
+- The framework provides unified resource usage and management capabilities, providing resource management capabilities for both the overall and individual backend tasks.
```

Comment:
```suggestion
- The framework provides unified resource usage and management capabilities for both overall and individual backend tasks.
```

Reply:
"provides unified resource usage and management capabilities" and "providing resource management capabilities" look a little duplicated.